sormas-backend/src/main/java/de/symeda/sormas/backend/travelentry/services/BaseTravelEntryService.java

```diff
@@ -87,4 +92,20 @@ protected Predicate createUserFilter(TravelEntryQueryContext qc) {
 	public List<TravelEntry> getAllAfter(Date date, Integer batchSize, String lastSynchronizedUuid) {
 		return getAllActiveAfter(date);
 	}
+
+	@Override
+	protected <T extends ChangeDateBuilder<T>> T addChangeDates(
+		T builder,
+		From<?, TravelEntry> travelEntryFrom,
+		boolean includeExtendedChangeDateFilters) {
+
+		builder = super.addChangeDates(builder, travelEntryFrom, includeExtendedChangeDateFilters);
+
+		if (includeExtendedChangeDateFilters) {
+			Join<TravelEntry, Case> resultingCase = travelEntryFrom.join(TravelEntry.RESULTING_CASE, JoinType.LEFT);
```

Comment:
I don't think a change to the case resulting from a travel entry should have any relevance for the change date of the travel entry itself; @MartinWahnschaffe What do you think?

Reply:
the evaluation of resulting case change date was removed from code